website/docs/fb/fbcode.md

```diff
@@ -22,8 +22,10 @@ Hydra includes buck TARGETS you can use in fbcode. In general, if there is TARGE
 1. You can depend on the TARGETS to use Hydra or a plugin.
 2. The TARGETS contains a runnable example.
 
+An example TARGET file can be accessed [here](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples/tutorials/basic/your_first_hydra_app/6_composition).
+
 targets are under `github/facebookresearch/hydra`:
-- `:hydra` : Primary target to use in most cases. Includes `hydra_oss` and the `fbcode_defaults`.
+- [:`hydra`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra) : Primary target to use in most cases. Includes `hydra_oss` and the `fbcode_defaults`.
 - `:hydra_oss` : Vanilla Hydra without any Facebook specific targets.
-- `plugins/...`: Plugins that have TARGETS file are runnable in fbcode.
-- `examples/...`: Examples that have a TARGETS file are runnable in `fbcode`. All tutorials (in `examples/tutorials`) are supported.
+- [`plugins`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/plugins): Plugins that have TARGETS file are runnable in fbcode.
+- [`examples`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples): Examples that have a TARGETS file are runnable in `fbcode`.
```

Comment:
are there any tutorials without a TARGETS file? why did you change the sentence?

Reply:
I think I saw something but now I cant seem to find it. Will revert this.